CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 - Use `SPLUNK_LISTEN_INTERFACE` and associated installer option to configure the network interface used by the collector for default configurations ([#3421](https://github.com/signalfx/splunk-otel-collector/pull/3421))
+  Existing installations will rely on the default value of `SPLUNK_LISTEN_INTERFACE` set to `0.0.0.0`. Users must add `SPLUNK_LISTEN_INTERFACE` to their collector configuration to take advantage of this new option.
```

Comment:
May be worth expanding this and moving to https://github.com/signalfx/splunk-otel-collector#advanced-configuration?

Reply:
Followed up with https://github.com/signalfx/splunk-otel-collector/pull/3510, thanks for the suggestion